docs/setup/custom-config.mdx

```diff
@@ -234,3 +267,118 @@ If this parameter is left empty (`[]`), users can upload files from any URL with
 The `web_crawling_allowed_sites` parameter restricts web crawling operations to a specified list of allowed IPs or web addresses. This ensures that the application only accesses pre-approved and safe URLs (`"web_crawling_allowed_sites": ["https://example.com", "https://api.mysite.com"]`).
 
 If left empty (`[]`), the application allows access to all URLs by default (marked with a wildcard in the open-source version).
+
+</Accordion>
+
+<Accordion title="Setup">
+
+First, create a `config.json` file.
+
+```bash
+{
+    "permanent_storage": {
+        "location": "local",
+        "bucket": "s3_bucket_name"
+    },
+    "paths": {
+        "root": "/home/mindsdb/var",
```

Comment:
user won't have 'mindsdb' folder. Do we need all following keys in this example?

Reply:
All following keys are optional. But you're right, I removed them from this example.

Reply:
I changed the path to `"root": "/path/to/root/location"`